.phpcs.xml.dist

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress IMporter">
+	<description>The code standard for the WordPress Importer plugin.</description>
+
+	<!-- What to scan -->
+	<file>.</file>
+	<exclude-pattern>vendor/</exclude-pattern>
+
+	<!-- How to scan -->
+	<arg value="sp"/> <!-- Show sniff and progress -->
+	<arg name="basepath" value="."/><!-- Strip the file paths down to the relevant bit -->
+	<arg name="colors"/>
+	<arg name="extensions" value="php"/>
+	<arg name="report" value="full"/>
+	<arg name="report" value="summary"/>
+	<arg name="report" value="source"/>
+
+	<!-- Rules: Check PHP version compatibility -->
+	<config name="testVersion" value="5.2-"/>
+	<rule ref="PHPCompatibility"/>
+
+	<!-- Rules: WordPress Coding Standards -->
+	<config name="minimum_supported_wp_version" value="3.6"/>
+	<rule ref="WordPress">
+		<exclude name="WordPress-VIP"/>
```

Comment:
Did you mean to do `<exclude name="WordPress-VIP"/>` ? Or did you intend it as `<exclude name="WordPress.VIP"/>` ?

Reply:
The latter - tried to add it back in manually.